pypi-dependencies.yaml

```diff
@@ -0,0 +1,251 @@
+build-commands: []
+buildsystem: simple
+modules:
+  - name: python3-deepl
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "deepl" --no-build-isolation
+    sources:
+      - &id008
+        type: file
+        url: https://files.pythonhosted.org/packages/71/4c/3db2b8021bd6f2f0ceb0e088d6b2d49147671f25832fb17970e9b583d742/certifi-2022.12.7-py3-none-any.whl
+        sha256: 4ad3232f5e926d6718ec31cfc1fcadfde020920e278684144551c91769c7bc18
+      - &id010
+        type: file
+        url: https://files.pythonhosted.org/packages/cc/f6/21a66e524658bd1dd7b89ac9d1ee8f7823f2d9701a2fbc458ab9ede53c63/charset_normalizer-3.1.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+        sha256: 75f2568b4189dda1c567339b48cba4ac7384accb9c2a7ed655cd86b04055c795
+      - &id011
+        type: file
+        url: https://files.pythonhosted.org/packages/90/00/eb918802fbf221456e817768a190c66b09ae607ccaa038c640a009e716e4/deepl-1.14.0-py3-none-any.whl
+        sha256: 7203b495d3356559b433544385d58c50257ad48035fc490730258b76fedd82f6
+      - &id014
+        type: file
+        url: https://files.pythonhosted.org/packages/fc/34/3030de6f1370931b9dbb4dad48f6ab1015ab1d32447850b9fc94e60097be/idna-3.4-py3-none-any.whl
+        sha256: 90b77e79eaa3eba6de819a0c442c0b4ceefc341a7a2ab77d7562bf49f425c5c2
+      - &id023
+        type: file
+        url: https://files.pythonhosted.org/packages/d2/f4/274d1dbe96b41cf4e0efb70cbced278ffd61b5c7bb70338b62af94ccb25b/requests-2.28.2-py3-none-any.whl
+        sha256: 64299f4909223da747622c030b781c0d7811e359c37124b4bd368fb8c6518baa
+      - &id027
+        type: file
+        url: https://files.pythonhosted.org/packages/7b/f5/890a0baca17a61c1f92f72b81d3c31523c99bec609e60c292ea55b387ae8/urllib3-1.26.15-py2.py3-none-any.whl
+        sha256: aa751d169e23c7479ce47a0cb0da579e3ede798f994f5816a74e4f4500dcea42
+  - name: python3-logzero
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "logzero" --no-build-isolation
+    sources:
+      - &id015
+        type: file
+        url: https://files.pythonhosted.org/packages/b3/68/aa714515d65090fcbcc9a1f3debd5a644b14aad11e59238f42f00bd4b298/logzero-1.7.0-py2.py3-none-any.whl
+        sha256: 23eb1f717a2736f9ab91ca0d43160fd2c996ad49ae6bad34652d47aba908769d
+  - name: python3-PySide6
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "PySide6" --no-build-isolation
+    sources:
+      - &id004
+        type: file
+        url: https://files.pythonhosted.org/packages/61/c1/37fabc94f958a03a605755ee1298d9f7d9f5d121f3d1ea2d480f79fc3d6c/PySide6-6.5.0-cp37-abi3-manylinux_2_28_x86_64.whl
```

Comment:
isn't this supposed to by Qt6?

Reply:
[PySide6](https://doc.qt.io/qtforpython-6/) is the Python module that offers the official Qt6 bindings for Python. So, not sure what you mean, sorry.

Reply:
`runtime-version: '5.15-22.08'`

This is Qt 5.15.